DataConnectors/NGINX/Connector_NGINX_agent.json

```diff
@@ -0,0 +1,149 @@
+{
+    "id": "NGINX HTTP Server",
+    "title": "NGINX HTTP Server",
+    "publisher": "Nginx",
+    "descriptionMarkdown": "NGINX HTTP Server data connector provides the capability to ingest [NGINX HTTP Server](https://nginx.org/en/) events into Azure Sentinel. Refer to [NGINX Logs documentation](https://nginx.org/en/docs/http/ngx_http_log_module.html) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **NGINXHTTPServer** in queries and workbooks. [Follow steps to get this Kusto Function>](https://github.com/Azure/Azure-Sentinel/blob/master/Parsers/NGINX/NGINXHTTPServer.txt)",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "NGINX Events",
+            "baseQuery": "NGINXHTTPServer"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Clients (Source IP)",
+            "query": "NGINXHTTPServer\n | summarize count() by SrcIpAddr\n | top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "NGINX_CL",
+            "lastDataReceivedQuery": "NGINXHTTPServer\n            | summarize Time = max(TimeGenerated)\n            | where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "NGINXHTTPServer\n            | summarize LastLogReceived = max(TimeGenerated)\n            | project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1
```

Comment:
isPreview tag is missing please refer sample template https://raw.githubusercontent.com/Azure/Azure-Sentinel/master/DataConnectors/Templates/Connector_REST_API_template.json

Reply:
updated